### ISSUE TITLE
[bug] Jphp 0.7

### ISSUE THREAD

**Issue by SerafimArts** [jphp-compiler/jphp#210]
This class: [WrapConfiguration.java](https://github.com/jphp-compiler/jphp/blob/master/jphp-runtime/src/php/runtime/ext/core/classes/util/WrapConfiguration.java) defined in the system as `Configuration`, not `php\util\Configuration`.

Example:
```php
$extensions = [];
foreach (get_loaded_extensions() as $ext) {
    $reflection = new ReflectionExtension($ext);
    foreach ($reflection->getClassNames() as $class) {
        echo $reflection->getName() . ': ' . $class . "\n";
    }
}
```

Outputs:
```
  ...
  ...
Core: php\util\Scanner
Core: php\util\Flow
Core: Configuration <-- Wrong namespace name
Core: php\util\Regex
Core: php\util\RegexException
  ...
  ...
```

**Comment by dim-s** [comment]
It's error in the reflection extension class, not in the Configuration class. Will be fixed.

**Comment by dim-s** [comment]
Fixed in 0.7.2.
